### DISCUSSION TITLE
If i have a private repository will people be able to view the site that is published from my repository?

### DISCUSSION THREAD

**Discussion by ShobanChiddarth** [community/community#22650] [Other Feature Feedback, Questions, & Ideas]
<p>Published in the settings(options)</p>

**Comment by techboyg5** [comment, marked as answer]
<p>If you’re talking about GitHub Pro, where you can publish a Pages site on a private repo, yes, anyone with the link can view your site.</p>
<p>However, it is entirely up to you if you want your site available to search engines.</p>
<p>If you don’t want your site available to search engines, then add the noindex meta tag into the head of all your pages:</p>
<pre><code class="lang-auto">&lt;meta name="robots" content="noindex"&gt;
</code></pre>
<p>And for GitHub Free, it is not possible to publish a Pages site on a private repo. All Pages sites must be on public repos.</p>
